app/deployments/tests/test_migrations.py

```diff
@@ -1,8 +1,10 @@
 from django.test import TestCase
+import pytest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pytest'](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=4987082)